client/ayon_deadline/plugins/publish/harmony/submit_harmony_deadline.py

```diff
@@ -269,6 +271,19 @@ def _unzip_scene_file(self, published_scene: Path) -> Path:
             self.log.error(published_scene)
             raise AssertionError("invalid scene format")
 
+        # TODO eventually replace with registered_host().work_root or similar
+        workdir = os.environ.get("AYON_WORKDIR")
+        if workdir and os.path.exists(workdir):
+            renders_path = os.path.join(workdir, "renders", "harmony")
+            os.makedirs(renders_path, exist_ok=True)
+
+            self.log.info(f"Copying '{published_scene}' -> '{renders_path}'")
+            shutil.copy(published_scene.as_posix(), renders_path)
+            published_scene = Path(
+                os.path.join(renders_path), published_scene.name)
+            (self._instance.context.data["cleanupFullPaths"].
+             append(published_scene))
```

Comment:
I'm very confused by this - why does the file need to be moved there? Does Harmony ALWAYS render next to the scene file?

Reply:
Actually - we are rolling our completely own AYONHarmony deadline plug-in which takes full control of the render. We don't need to change anything, but should just update how the deadline rendering behaves and renders in that plug-in then? We could e.g. pass it a "Output Path" that _is_ the expected output path, we don't need to move any files?

Reply:
It seems we should not be moving anything - but should just be changing the `render_path` logic in the `get_plugin_info` call, right?

Reply:
You (DL) need to have project unzipped, someone needs to do it. 
AYON DL plugin is there only because official plugin returned exit code 1 at end of rendering for some reason(I think). That could be revisited if it still needed.
We could probably move unzipping workfile to AYON DL Plugin that way each worker would download and unzip workfile to their temp folder, but I don't think we should be doing it. I would prefer to get rid of the AYON specific DL plugin altogether.

Reply:
Ah totally forgot we're dealing with workfiles that aren't actually workfiles for the host application. 👍